### DISCUSSION TITLE
Is it possible to make Rowy use snake_case for field keys (by default) instead of camelCase?

### DISCUSSION THREAD

**Discussion by harinij** [AntlerVC/firetable#749] [Support / Q&A] [answered]
Reposting question from Discord.

```
Is it possible to make Rowy use snake_case for field keys (by default) instead of camelCase? 
FlutterFlow's default when it generates a user collection is snake case.
```

**Comment by harinij** [comment, marked as answer]
By default the suggested field name for columns is camelCase however it is editable you can update it to snake_case. As showcased in the video below.

https://user-images.githubusercontent.com/307298/185339318-3382db8a-3947-4ec6-8626-9c372a6b2c70.mov

**Reply by tnypxl** [reply]
Thank you @harinij!
